_static/style.css

```diff
@@ -109,6 +109,9 @@ blockquote {
     margin-bottom: 10px;
 }
 
+code {
+    margin-left: 1.5em;
+}
```

Comment:
Not needed if using the `.. code::` blocks.

Reply:
Ah! I will change it...